### PR TITLE
Add hoodie to the other solutions comparison matrix (fixes #282)

### LINKED ISSUE #282: Add hood.ie to the other solutions comparison matrix
In the [comparison matrix with other solutions](http://kinto.readthedocs.org/en/latest/overview.html#comparison-with-other-solutions), we should probably add an entry for [hoodie](http://docs.hood.ie/en/).

### PR DESCRIPTION
**Pull request by niraj8**
I tried to find about the features Hoodie has currently. I couldn't find if they support pluggable storage and decentralised discovery.

### CODE REVIEW

**Comment by leplatrem** [comment]
This looks valid to me. @n1k0 @almet any feedback ?

I think we might want to precise that Hoodie is using PouchDB (and thus CouchBD)

**Comment by n1k0** [comment]
Yeah, this looks good.

**Comment by Natim** [comment]
Thanks a lot :+1:

File: docs/overview.rst [hunk 1]
```diff
@@ -119,25 +119,26 @@ What follows is a comparison table showing how Kinto stacks up compared to some
...
+Revision history                                       ✔                        ✔
+File storage                         ✔                 ✔        ✔               ✔
+Batch/bulk operations        ✔       ✔                 ✔                        ✔
+Changes stream               [#]_    ✔       ✔         ✔                        ✔
+Pluggable authentication     ✔                         ✔        [#]_            ✔
+Pluggable storage / cache    ✔                                  ✔              
+Self-hostable                ✔                         ✔        ✔               ✔
+Decentralised discovery      [#]_                               ✔               
+Open source                  ✔                         ✔        ✔               ✔
+Language                     Python                    Erlang   Node.js [#]_    Node.js
```
Comment (almet) [inline docs/overview.rst]: Is Hoodie really Node.js? I thought it was Erlang as well (for Couch)

Comment (niraj8) [inline docs/overview.rst]: Yes. screenshot 2015-12-11 14 52 00

http://docs.hood.ie/en/start/

**Pull request merged**